edx_rest_framework_extensions/utils.py

```diff
@@ -1,12 +1,26 @@
 """ Utility functions. """
 import logging
-
```

Comment:
Revert. `jwt` is not a default Python library.

Imports should be grouped as follows: Python core, third-party, local